integration-tests/jpa-mysqldb/README.md

```diff
@@ -0,0 +1,30 @@
+# JPA example with MySQLDB
+
+## Running the tests
+
+By default, the tests of this module are disabled.
+
+To run the tests in a standard JVM with MySQLDB started as a Docker container, you can run the following command:
+
+```
+mvn clean install -Dtest-mysqldb -Ddocker
+```
+
+Please note that waiting on the availability of MySQLDB port does not work on macOS.
```

Comment:
FYI the wait condition isn't working on Linux either. I remember having a lot of trouble with Mariadb as well: the database opens the port way before it's actually ready.

I suspect we'll either need to find a different way to test these on CI; some possibilities:
 
1. having an advance prepared MySQL instance to connect to
2. find a different waiting condition
3. create a custom container image which makes it easier to wait for its readiness state

Reply:
I opted for the first proposal.